sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -970,6 +986,21 @@ public PDone apply(PCollection<TableRow> input) {
         return VoidCoder.of();
       }
 
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        if (table != null) {
+          builder.add("tableSpec", toTableSpec(table));
+        }
+
+        if (schema != null) {
+          builder.add("schema", schema.toString());
```

Comment:
There's been some discussion here. Prettification would definitely be useful, but it's not on our immediate plans. We'll likely look at usage to determine what "prettification" to prioritize (syntax highlighting, json formatting, line wrapping, etc.)  /cc: @bjchambers @ajamato